pages/sites/[slug]/complete-signup.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import CompleteSignup from '../../../src/features/user/CompleteSignup';
+import Head from 'next/head';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+import {
+  constructPathsForTenantSlug,
+  getTenantConfig,
+} from '../../../src/utils/multiTenancy/helpers';
+import { Tenant } from '@planet-sdk/common/build/types/tenant';
+import { useRouter } from 'next/router';
+import { useTenant } from '../../../src/features/common/Layout/TenantContext';
+import {
+  GetStaticProps,
+  GetStaticPropsContext,
+  GetStaticPropsResult,
+} from 'next';
+import { defaultTenant } from '../../../tenant.config';
+
+interface Props {
+  pageProps: {
+    tenantConfig: Tenant;
+  };
+}
+
+export default function UserProfile({ pageProps }: Props) {
+  const router = useRouter();
+  const { setTenantConfig } = useTenant();
+
+  React.useEffect(() => {
+    if (router.isReady) {
+      setTenantConfig(pageProps.tenantConfig);
+    }
+  }, [router.isReady]);
+
+  return pageProps.tenantConfig ? (
+    <>
+      <Head>
+        <title>{`${pageProps.tenantConfig.config.meta.title} - Complete SignUp`}</title>
+      </Head>
+      <CompleteSignup />
+    </>
+  ) : (
+    <></>
+  );
+}
+
+export async function getStaticPaths() {
+  return {
+    paths: await constructPathsForTenantSlug(),
+    fallback: 'blocking',
+  };
+}
+
+interface StaticProps {
+  tenantConfig: Tenant;
+}
+
+export const getStaticProps: GetStaticProps<StaticProps> = async (
+  context: GetStaticPropsContext
+): Promise<GetStaticPropsResult<StaticProps>> => {
+  const tenantConfig =
+    (await getTenantConfig(context.params?.slug as string)) ?? defaultTenant;
+
+  return {
+    props: {
+      ...(await serverSideTranslations(
+        context.locale || 'en',
+        [
+          'bulkCodes',
+          'common',
+          'country',
+          'donate',
+          'donationLink',
+          'editProfile',
+          'giftfunds',
+          'leaderboard',
+          'managePayouts',
+          'manageProjects',
+          'maps',
+          'me',
+          'planet',
+          'planetcash',
+          'redeem',
+          'registerTrees',
+          'tenants',
+          'treemapper',
+        ],
```

Comment:
This list was reduced to ['common', 'country', 'editProfile'], and has been inadvertently updated to include all translation files. Should be reverted.